src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -89,7 +89,7 @@ const overviewIDsToHide = ['kibanaOverview', 'enterpriseSearch'];
 const overviewIDs = [
   ...overviewIDsToHide,
   'observability-overview',
-  'securitySolutionUI:overview',
+  'securitySolutionUI:get_started',
```

Comment:
I am not sure about this one, is this change to hide the "Get Started" item from the Security section in the global nav?